Makefile

```diff
@@ -27,11 +28,16 @@ TEST_COVER_PACKAGES:=go list ./pkg/... | grep -vE "pkg/client" | grep -vE "pkg/t
 default: build
 
 docker-push: docker backup-docker
-	docker push "${DOCKER_REGISTRY}/pingcap/tidb-operator:latest"
-	docker push "${DOCKER_REGISTRY}/pingcap/tidb-backup-manager:latest"
+	docker push "${DOCKER_REGISTRY}/pingcap/tidb-operator:${IMAGE_TAG}"
+	docker push "${DOCKER_REGISTRY}/pingcap/tidb-backup-manager:${IMAGE_TAG}"
 
+ifeq ($(NO_BUILD),y)
```

Comment:
NO_BUILD is added to skip building binaries because in e2e we use pre-built binaries from the previous stage